cmd/frontend/graphqlbackend/external_services.go

```diff
@@ -116,6 +121,10 @@ type updateExternalServiceInput struct {
 }
 
 func (r *schemaResolver) UpdateExternalService(ctx context.Context, args *updateExternalServiceArgs) (*externalServiceResolver, error) {
+	start := time.Now()
+	var err error
+	namespaceUserID, namespaceOrgID := int32(0), int32(0)
```

Comment:
I think this would do the same and would look more idiomatic to me:
```suggestion
	var namespaceUserID, namespaceOrgID int32
```